brave/src/main/java/brave/internal/Platform.java

```diff
@@ -175,63 +172,81 @@ static Jre9 buildIfSupported(boolean zipkinV1Present) {
       };
     }
 
-    @IgnoreJRERequirement
-    @Override public long randomLong() {
-      return java.util.concurrent.ThreadLocalRandom.current().nextLong();
+    Jre9(boolean zipkinV1Present) {
+      super(zipkinV1Present);
     }
 
-    @IgnoreJRERequirement
-    @Override public long nextTraceIdHigh() {
-      return nextTraceIdHigh(java.util.concurrent.ThreadLocalRandom.current());
```

Comment:
using a static method like this indirectly caused the code to compile against the TLR type, subverting the attempt to defer usage until runtime. I fixed the signature so that it couldn't do this anymore.